Python/Storage/blobxfer.py

```diff
@@ -304,7 +306,7 @@ def get_blob_properties(self, container_name, blob_name):
 
     def put_blob(
             self, container_name, blob_name, blob, x_ms_blob_type,
-            x_ms_blob_content_md5, x_ms_blob_content_length):
+            x_ms_blob_content_md5, x_ms_blob_content_length, x_ms_blob_content_type="application/octet-stream"):
```

Comment:
Wrong parameter order, this must match the azure storage sdk ordering (place before md5). Also, do not have a default of a string, default should be None.

Additionally make sure your code is flake8 clean.